sbnanaobj/StandardRecord/SRBNBInfo.cxx

```diff
@@ -26,12 +26,11 @@ namespace caf
     M876BB_spill_time_diff(std::numeric_limits<float>::signaling_NaN()),
     MMBTBB_spill_time_diff(std::numeric_limits<float>::signaling_NaN())
   {}
-  SRBNBInfo::~SRBNBInfo() {}
 
   void SRBNBInfo::setDefault()
   {
-    spill_time_sec = 0;
-    spill_time_nsec = 0;
+    spill_time_sec = UINT_MAX;
+    spill_time_nsec = UINT_MAX;
```

Comment:
`spill_time_sec` and `spill_time_nsec` are of type `long`, their `MAX` value should reflect that (I suspect that `long` and `int` in our architecture happen to be of the same size, but that is not mandated).
C:
```suggestion
    spill_time_sec = ULONG_MAX;
    spill_time_nsec = ULONG_MAX;
```
or C++:
```suggestion
    spill_time_sec = std::numeric_limits<unsigned long int>::max();
    spill_time_nsec = std::numeric_limits<unsigned long int>::max();
```
(the latter requires `#include <limits>`)